lib/services/northBound/contextServer.js

```diff
@@ -39,7 +39,8 @@ var async = require('async'),
     context = {
```

Comment:
An entry corresponding to the work done in this PR should be added to CHANGES_NEXT_RELEASE file, referring issue #527

Reply:
Fixed with 82144cbd43599aecebddf0037bc85f270942db5f